src/components/pages/profile/[name]/tabs/MoreTab/Miscellaneous.tsx

```diff
@@ -164,71 +195,199 @@ const Miscellaneous = ({
 
   const makeEvent: () => CalendarEvent = useCallback(
     () => ({
-      title: `Renew ${name}`,
+      title: t('name.renew', { name }),
       start: expiryDate,
       duration: [10, 'minute'],
       url: window.location.href,
     }),
-    [name, expiryDate],
+    [name, expiryDate, t],
   )
 
+  const {
+    register,
+    handleSubmit,
+    setError,
+    clearErrors,
+    formState: { errors, isSubmitting },
+  } = useForm<FormData>({ mode: 'onChange' })
+
+  const onSubmit = async ({ email }: FormData) => {
+    try {
+      const response = await fetch(EARNIFI_ENDPOINT, {
+        method: 'POST',
+        headers: {
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          email,
+          address: name,
+          chainId,
+        }),
+      })
+      if (response?.ok) {
+        setHasCreatedReminder(true)
+        return
+      }
+      if (response?.status === 400) {
+        const responseMessage = ((await response.json()) as any)?.message
+        setError('email', {
+          type: 'submitError',
+          message: responseMessage || t('tabs.more.misc.earnfi.submitError', { ns: 'profile' }),
+        })
+        return
+      }
+      throw new Error('Submission error')
+    } catch (e) {
+      setError('email', {
+        type: 'submitError',
+        message: t('tabs.more.misc.earnfi.submitError', { ns: 'profile' }),
+      })
+    }
+    setTimeout(() => {
+      clearErrors('email')
+    }, 3000)
+  }
+
   if (!expiryDate) return null
 
   return (
-    <MiscellaneousContainer $isCached={isCachedData || registrationCachedData}>
-      <DatesContainer>
-        {registrationData && (
+    <>
+      <Dialog open={hasEarnifiDialog} variant="blank" onDismiss={() => null}>
+        <Dialog.Heading title={t('tabs.more.misc.earnfi.title', { ns: 'profile' })} />
+        {match(hasCreatedReminder)
+          .with(false, () => (
+            <Form onSubmit={handleSubmit(onSubmit)}>
+              <Typography style={{ textAlign: 'center' }}>
+                <Trans
+                  style={{ textAlign: 'center' }}
+                  i18nKey="tabs.more.misc.earnfi.enterEmail"
+                  ns="profile"
+                  components={{
+                    a: <Outlink href={EARNIFI_OUTLINK} />,
+                  }}
+                />
+              </Typography>
+              <Spacer $height="3" />
+              <Input
+                type="email"
+                id="email"
+                label={t('action.enterEmail')}
+                {...register('email', {
+                  required: t('errors.emailRequired'),
+                  pattern: {
+                    value: emailRegex,
```

Comment:
can native email validation be used? (https://developer.mozilla.org/en-US/docs/Learn/Forms/HTML5_input_types#email_address_field)